changes/11826-chromeos-support-dashboard

```diff
@@ -0,0 +1,3 @@
+- Added built-in label for ChromeOS.
+- Fixed issue with `chrome_extensions` virtual table not returning path column on `fleetd-chrome`.
```

Comment:
Let's clarify that now it is returning empty column to not break software ingestion.

Reply:
9450bec34d9110d4965927f7b00540ef061be355